python/paddle/fluid/tests/unittests/test_lookup_table_op.py

```diff
@@ -64,6 +64,7 @@ def test_check_output(self):
         self.attrs = {'padding_idx': int(padding_idx)}
         self.check_output()
 
+    @skip_check_grad_ci
```

Comment:
How about the followings? i.e remove line68-71
```
@skip_check_grad_ci(note)
class TestLookupTableOpWithPadding(TestLookupTableOp):
   def setUp(self):
     xxx
   def test_check_output(self):
     xxx
```

Reply:
Done.